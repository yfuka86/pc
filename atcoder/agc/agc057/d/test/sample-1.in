13
3 1
3 2
7 1
7 2
7 3
7 4
10 1
10 2
10 3
10 4
10 5
2022 507
1000000000000000000 999999999999999999
