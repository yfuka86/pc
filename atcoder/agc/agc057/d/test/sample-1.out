2
-1
2
4
6
-1
3
6
8
9
-1
1351
-1
